MVVM Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -66,7 +66,7 @@ private class FeedLoaderStub: FeedLoader {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		(tableView.tableHeaderView as? ErrorView)?.show(message: errorMessage)
```

Comment:
Same here - Make sure to decouple the test from the view hierarchy so changing the hierarchy won't break the tests.

```suggestion
		errorView?.show(message: errorMessage)
```

Reply:
yea, I see your point 👍